04-make_epochs.py

```diff
@@ -57,6 +55,8 @@ def run_epochs(subject):
     # Construct metadata from the epochs
     # Add here if you need to attach a pandas dataframe as metadata
     # to your epochs object.
+    
+    # XXX: nice example missing
```

Comment:
@agramfort: I remember I used an example on how to implement metadata, but could not find it. Do you have a suggestion to what we could point?

Reply:
https://martinos.org/mne/dev/auto_tutorials/plot_metadata_epochs.html